src/utils/process-get-filters.js

```diff
@@ -1,36 +1,42 @@
 import _ from 'lodash';
-/**
- * Removes the "filter[]" wrapper from a filter parameter
- *
- * @param {string} filterName
- * @returns {string} the filter name with 'filter' and brackets removed
- */
-const normalizeFilterName = (filterName) => {
-  const filterTextRegex = /filter\[(.*)\]/g;
-  const regexResults = filterTextRegex.exec(filterName);
-  return regexResults ? regexResults[1] : filterName;
-};
-
-/**
- * Returns filters with keys changed to remove the "filter[]" wrapper
- *
- * @param {object} filters
- * @returns {object}
- */
-const normalizeFilterNames = (filters) => _.mapKeys(filters,
-  (filterValue, filterName) => normalizeFilterName(filterName));
 
 class GetFilterProcessor {
+  /**
+   * Returns filters with keys changed to remove the "filter[]" wrapper
+   *
+   * @param {object} filters
+   * @returns {object}
+   */
+  static normalizeFilterNames(filters) {
+    return _.mapKeys(filters,
+      (filterValue, filterName) => GetFilterProcessor.normalizeFilterName(filterName));
+  }
+
+  /**
+   * Removes the "filter[]" wrapper from a filter parameter
+   *
+   * @param {string} filterName
+   * @returns {string} the filter name with 'filter' and brackets removed
+   */
+  static normalizeFilterName(filterName) {
+    const filterTextRegex = /filter\[(.*)\]/g;
+    const regexResults = filterTextRegex.exec(filterName);
+    return regexResults[1] || filterName;
+  }
+
   /**
    * Initializes a new filter process using the parameters from the
    * get request for the resource and a map of column names to resource attribute names
-   * @param {array} getParameters
+   *
+   * @param {Array} getParameters
```

Comment:
Array of?

Reply:
changed => `Array<object>`, would be handy to have typescript types here :/

Reply:
And it should be `object[]`: https://github.com/osu-mist/students-api/blob/207194b7d9fa346494972ef2fc14773f8e72ee7b/src/utils/jsonapi.js#L20

Reply:
this should be actually fixed and the commits should be pushed now